pkg/webhook/webhook_roles.go

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package webhook
+
+import (
+	"context"
+
+	"github.com/devfile/devworkspace-operator/webhook/server"
+	v1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func CreateWebhookRole(client crclient.Client,
```

Comment:
```suggestion
func CreateWebhookClusterRole(client crclient.Client,
```
And a similar rename for the filename